core/src/main/scala/org/apache/spark/deploy/LocalSparkCluster.scala

```diff
@@ -17,10 +17,13 @@
 
 package org.apache.spark.deploy
 
+import java.net.InetAddress
+
 import scala.collection.mutable.ArrayBuffer
 
 import akka.actor.ActorSystem
 
+import com.google.common.net.InetAddresses
```

Comment:
Tiny nit: this import can be grouped with the `akka` one above instead of Spark imports.

I like this change. How about the use of `Utils.localIpAddress` in `HttpServer`? It might need a similar treatment, I don't know.